### PR TITLE
[WIP] Update Quickstart to use wheelhouse config from yaml

### PR DESCRIPTION
**Pull request by mshriver**
@RonnyPfannschmidt will definitely be interested in this one I think :)

So I'd like to add wheelhouse support to quickstart in order to support CI setup+testing via miq-runtest, and I'm trying by looking for a wheelhouse config in the cfme_data yaml file. This requires including yaycl import, so I'm adding a yaycl wheel file for the script, using distro import as an example.

The script should cleanly run if no wheelhouse is spec'd in the yaml, or if the yaml isn't found.

Include --trusted-host and -f options to pip, from `basic_info.wheelhouse.host` and `basic_info.wheelhouse.url`, thus we don't have to try and parse a host from only a URL.

### CODE REVIEW

File: cfme/scripting/quickstart.py [hunk 1]
```diff
@@ -163,12 +194,16 @@ def install_requirements(venv_path, quiet=False):
...
 
+    wheelhouse = get_wheelhouse(config_path)
+
     venv_call(
         venv_path,
         'pip', 'install',
         '-r', REQUIREMENT_FILE,
         '--no-binary', 'pycurl',
-        *(['-q'] if quiet else []))
+        ('-q' if quiet else ''),
```
Comment (mshriver) [inline cfme/scripting/quickstart.py]: This I haven't tested yet, might have to play with the args/kwargs. python <3.5 doesn't support multiple *[] references in method calls, might have to combine -q and -f args logic here.

**Comment by mshriver** [comment]
Still WIP, but tagging @psav and @RonnyPfannschmidt for a review to make sure I'm not going off the deep end here.

**Comment by RonnyPfannschmidt** [comment]
at first glance, i think this place ain't the right one for that change - in #5317 i just got rid of putting wheels into the repo as well

we should have a discussion soon

**Comment by mshriver** [comment]
Going to try this a better way, as suggested by @RonnyPfannschmidt. WIll re-open if need be.

**Pull request closed without merging**